packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -14,19 +14,29 @@ import uuid from 'uuid';
 import {InvalidUserCreation} from './convo-error';
 import ShareActivity from './share-activity';
 
+const idToUrl = new Map();
 
 const Conversation = WebexPlugin.extend({
   namespace: 'Conversation',
 
+  /**
+   * @param {Object} conversation
+   * @param {Object} object
+   * @param {Object} activity
+   * @returns {Promise}
+   */
   acknowledge(conversation, object, activity) {
+    const url = this.getConvoUrl(conversation);
+    const convoWithUrl = Object.assign({}, conversation, {url});
```

Comment:
Sorry, I didn't notice this before, but it seems like the `convoWithUrl` is used a lot right before calling `prepareConversation`. Does it make sense to move the `getConvoUrl` to within `prepareConversation`?

Reply:
I was trying to keep the lines short but I can move it within `convoWithUrl`